app/javascript/components/supplemental_application/SupplementalApplicationContainer.js

```diff
@@ -39,75 +39,63 @@ const StatusUpdateSection = withContext(({ store, formIsValid }) => {
   )
 })
 
-const ConfirmedPreferencesSection = ({ application, applicationMembers, fileBaseUrl, onSave, confirmedPreferencesFailed, onDismissError, form }) => {
-  return (
-    <ContentSection
-      title='Confirmed Preferences'
-      description='Please allow the applicant 24 hours to provide appropriate preference proof if not previously supplied.'>
-      <ContentSection.Content>
-        { confirmedPreferencesFailed && (
-          <AlertBox
-            invert
-            onCloseClick={onDismissError}
-            message="We weren't able to save your updates. Please try again." />
-        )}
-        <PreferencesTable
-          application={application}
-          applicationMembers={applicationMembers}
-          onSave={onSave}
-          fileBaseUrl={fileBaseUrl}
-          onPanelClose={onDismissError}
-          form={form}
-        />
-      </ContentSection.Content>
-    </ContentSection>
-  )
-}
-
-const ConfirmedHousehold = ({ amis, amiCharts, form }) => {
-  return (
-    <ContentSection title='Confirmed Household'>
-      <ContentSection.Sub title='Confirmed Reserved and Priority Units'>
-        <ConfirmedUnits />
-      </ContentSection.Sub>
-      <ContentSection.Sub title='Confirmed Household Income'>
-        <ConfirmedHouseholdIncome amis={amis} amiCharts={amiCharts} form={form} />
-      </ContentSection.Sub>
-    </ContentSection>
-  )
-}
-
-const LeaseInformationSection = ({form}) => {
-  return (
-    <ContentSection title='Lease Information'>
-      <ContentSection.Content borderBottom>
-        <LeaseInformationInputs form={form} />
-      </ContentSection.Content>
-    </ContentSection>
-  )
-}
+const ConfirmedPreferencesSection = ({ application, applicationMembers, fileBaseUrl, onSave, confirmedPreferencesFailed, onDismissError, form }) => (
```

Comment:
any idea why the diff is so wonky on this file? It's kinda hard to see what the actual changes are

Reply:
looks like it's mostly just removing returns?

Reply:
Yea just doing the direct returns moves everything over by 2 spaces